src/open_inwoner/templates/pages/kvk/branches.html

```diff
@@ -2,36 +2,106 @@
 {% load i18n static form_tags grid_tags button_tags card_tags button_tags %}
 
 {% block content %}
-<p> {% trans "Selecteer de vestiging waarmee u wilt inloggen" %}:</p>
-{% render_grid %}
-    {% render_column start=0 span=6 %}
-        <form method="POST" id="eherkenning-branch-form">
-            {% csrf_token %}
-            {% if form.branch_number.errors %}
-                {% errors errors=form.branch_number.errors %}
-            {% endif %}
-            {% for branch in company_branches %}
-                {% render_card direction='horizontal' %}
+    <h1 class="h1" id="title">{% trans "Log in" %}</h1>
+    {% render_grid %}
+        {% render_column start=4 span=6 %}
+            <form method="POST" id="eherkenning-branch-form" class="form">
+                {% csrf_token %}
+                {% if form.branch_number.errors %}
+                    {% errors errors=form.branch_number.errors %}
+                {% endif %}
+
+                <ul class="choice-list choice-list--single">
+
+                {# if COMPANY shows NEVENVESTIGING and HOOFDVESTIGING in one list #}
+                <p><strong>{% trans "Selecteer de vestiging waarmee u wilt inloggen" %}</strong></p>
+                {% for branch in company_branches %}
                     {% with company_id=branch.vestigingsnummer %}
-                        <label class="left" for="branch-{{ company_id }}">
-                            <p>{{ branch.naam }}</p>
-                            {% if branch.straatnaam and branch.plaats %}
-                                <p>{{ branch.straatnaam }}, {{ branch.plaats}}</p>
-                            {% endif %}
-                            {% if branch.vestigingsnummer %}
-                                <p>{% trans "Vestiging" %}: {{ company_id }}</p>
-                            {% endif %}
-                        </label>
-                        <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}">
+
+                        <li class="choice-list__item">
+                            <label class="choice-list__label">
+                                <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}" class="choice-list__radio">
+                                <div class="choice-list__content">
+                                    <h2 class="choice-list__heading">{{ branch.naam }}</h2>
```

Comment:
The check seems to be not completely centered (on Firefox):
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/0ae164a6-ed70-4843-997f-7627746e2d89)

compared to the design:
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/d8d7bfd0-cade-41ee-b956-aab55ce4f6dd)

Reply:
I've done an fix for Windows machines with line-height; let's see if this looks better on Linux now too.

Reply:
Looks good :+1: 
![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/2043fa9a-745e-4281-96a2-16126ff5f442)